storage/block/api-cli/creating-a-volume.mdx

```diff
@@ -21,7 +21,7 @@ dates:
 
 </Message>
 
-1. Create a block volume from the API by launching the following command:
+1. Create a block volume from your API by launching the following command:
```

Comment:
Pourquoi your?

Reply:
agree, I think "the" makes more sense here :-)

Reply:
Thank you, I changed it back to "the"!